src/adagrams.js

```diff
@@ -1,15 +1,125 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const distOfLetters = {
+    A: 9,
+    B: 2,
+    C: 2,
+    D: 4,
+    E: 12,
+    F: 2,
+    G: 3,
+    H: 2,
+    I: 9,
+    J: 1,
+    K: 1,
+    L: 4,
+    M: 2,
+    N: 6,
+    O: 8,
+    P: 2,
+    Q: 1,
+    R: 6,
+    S: 4,
+    T: 6,
+    U: 4,
+    V: 2,
+    W: 2,
+    X: 1,
+    Y: 2,
+    Z: 1,
+  };
+  
+  const availableLetters = [];
+  const randomDrawnLetters = [];
+
+  for (const letter in distOfLetters) {
+    for (let i = 0; i < distOfLetters[letter]; i++) {
+        availableLetters.push(letter)
+    }
```

Comment:
Nice! Could also use `Object.entries`, like:

```suggestion
  for (const [letter, letterFrequency] of Object.entries(distOfLetters)) {
    for (let i = 0; i < letterFrequency; i++) {
      availableLetters.push(letter);
    }
```